internal/configtxgen/encoder/encoder.go

```diff
@@ -206,6 +211,37 @@ func NewOrdererGroup(conf *genesisconfig.Orderer) (*cb.ConfigGroup, error) {
 		if consensusMetadata, err = channelconfig.MarshalEtcdRaftMetadata(conf.EtcdRaft); err != nil {
 			return nil, errors.Errorf("cannot marshal metadata for orderer type %s: %s", ConsensusTypeEtcdRaft, err)
 		}
+	case ConsensusTypeSmartBft:
+		var consenterProtos []*cb.Consenter
+		for _, consenter := range conf.ConsenterMapping {
+			c := &cb.Consenter{
+				Id:       consenter.ID,
+				Host:     consenter.Host,
+				Port:     consenter.Port,
+				MspId:    consenter.MSPID,
+				Identity: consenter.Identity,
```

Comment:
shouldn't we do some similar file loading as we do for TLS certs, for identity as well?